packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -28,18 +39,48 @@ const selectTargetAccountPrompt = async (
         };
       });
   } else {
+    let sandboxUsage = {};
+    try {
+      const accountId = getAccountId(defaultAccountConfig.portalId);
+      sandboxUsage = await getSandboxUsageLimits(accountId);
+    } catch (err) {
+      logger.debug('Unable to fetch sandbox usage limits: ', err);
+    }
+    const sandboxAccounts = accounts.reverse().filter(isSandbox);
+    const createSandboxDisabled =
```

Comment:
I don't think we need `createSandboxDisabled` if we're also making the same checks on lines 54 and 59-60 right?